public_html/wp-content/plugins/camptix/addons/require-login.php

```diff
@@ -59,16 +56,37 @@ public function __construct() {
 	public function block_unauthenticated_actions() {
 		/** @var $camptix CampTix_Plugin */
 		global $camptix;
+		// Continue normal request, this is not a tickets page.
+		if ( ! isset( $_REQUEST['tix_action'] ) ) {
+			return;
+		}
+
```

Comment:
I changed the logic a little here so we don't need to keep checking for this being set.